README.md

```diff
@@ -24,13 +24,20 @@ var builder = searchable.Builder{
 }
 
 func main() {
-  search := builder.Search([]string{"alice", "45", "-admin"})
+  search := builder.Search([]searchable.Token{
+    {Term: "alice"},
+    {Term: "45"},
+    {Term: "admin", Negate: true},
+  })
   users := squirrel.Select("*").From("users").Where(search)
   sql, args, _ := users.ToSql()
 
   fmt.Println(sql)
   // => SELECT * FROM users WHERE (((users.name IS NOT NULL AND users.name LIKE ?) OR (users.code IS NOT NULL AND users.code = ?)) AND ((users.name IS NOT NULL AND users.name LIKE ?) OR ...
   fmt.Println(args)
   // => [%alice% alice %45% 45 45 %admin% admin]
+
+  // for a simple search of strings without requirement to negate terms, use SearchStrings()
+  search_string := builder.SearchStrings([]string{"alice", "45", "admin"})
```

Comment:
soo, pls remove that, it's not really a "complete" example, let's add a separate PR with examples instead (something that Go supports natively, I will explain)

Reply:
Go has the ability to add examples as tests, see https://blog.golang.org/examples. You can e.g. (separate PR pls) add an examples_test.go where you document the various method, e.g.:

```
func ExampleBuilder_SearchStrings() {
  builder := searchable.Builder{
    {SQL: "users.name"},
    {SQL: "users.age", Type: searchable.TypeInt},
    {SQL: "users.code", Exact: true},
  }
  clause := builder.SearchStrings([]string{"alice", "45", "admin"})
  fmt.Printf("%v\n", clause)

  // Output:
  // Expected output goes here!
}
```